benchmark_runner/grafana/perf/jsonnet/g.libsonnet

```diff
@@ -0,0 +1 @@
+import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet'
```

Comment:
Is there something special about g.libsonnet that you can't just have something else use main.gsonnet?

Reply:
All is according to [grafonnet](https://github.com/grafana/grafonnet/blob/main/examples/terraform/jsonnet/g.libsonnet)